src/test/java/com/google/sps/PlacesTest.java

```diff
@@ -0,0 +1,51 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+import com.google.maps.model.PlacesSearchResult;
+import com.google.sps.data.Places;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public final class PlacesTest {
+  @Test
+  public void getNonEmptyArrayListOfPlaces() {
+    ArrayList<PlacesSearchResult[]> places = Places.getAllPlaces();
+    assertFalse(places.isEmpty());
+  }
+
+  @Test
+  public void getUpToSixListsOfPlacesResults() {
+    ArrayList<PlacesSearchResult[]> places = Places.getAllPlaces();
+    assertTrue(places.size() <= 6);
+  }
+
+  @Test 
+  public void getDifferentPlacesSearchResults() {
+    ArrayList<PlacesSearchResult[]> places = Places.getAllPlaces();
+    for (int i = 0; i < places.size() - 1; i++) {
+      assertFalse(Arrays.equals(places.get(i), places.get(i + 1)));
+    }
```

Comment:
Just checking, is it possible for the call to return duplicate results? if duplicates are possible, could they potentially be at non-adjacent indexes? places.get(i) potentially be equal to places.get(i+5)?

Reply:
I decided to change the collection to a set to avoid duplicates